terraform/modules/data-share-service/prometheus.tf

```diff
@@ -11,11 +11,47 @@ resource "random_password" "prometheus_password" {
   special = false
 }
 
-data "aws_iam_policy" "grafana_prometheus_access" {
-  name = "AmazonPrometheusQueryAccess"
+data "aws_iam_policy_document" "grafana_prometheus_access" {
+  statement {
+    actions = [
+      "aps:GetLabels",
+      "aps:GetMetricMetadata",
+      "aps:GetSeries",
+      "aps:QueryMetrics",
+      "aps:ListRules",
+      "aps:ListAlertManagerSilences",
+      "aps:ListAlertManagerAlerts",
+      "aps:GetAlertManagerStatus",
+      "aps:ListAlertManagerAlertGroups",
+      "aps:PutAlertManagerSilences",
+      "aps:DeleteAlertManagerSilences",
```

Comment:
Does this mean that we _can_ set up AMP alert manager as a datasource in grafana?

Reply:
Yep :)